workout-tracker/src/components/Button/Button.css

```diff
@@ -0,0 +1,137 @@
+.button {
```

Comment:
Looks good man but are we not using a bootstrapped styling theme like the ones you showed us Or this is just a temp replacement, but overall good one

Reply:
I popped in this reusable component because it's quite versatile (can be used as a react-router link or a button). In V3 we can refactor to design systems if we want.